src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -269,6 +269,32 @@ function Restore() {
   }
 }
 
+function KeylessBackupSetupContainer({
+  isOnboarding,
+  title,
+  subtitle,
+  children,
+}: {
+  isOnboarding: boolean
+  title: string
+  subtitle: string
+  step: number
+  totalSteps: number
```

Comment:
seems like these 2 are not used. Can we either drop these or drop title / subtitle and construct the header from it (probably more DRY)

Reply:
Removed the props in 5402d6da01b077fb88a9c3cab431dad4a2d4d657!